treeherder/perf/auto_perf_sheriffing/backfill_tool.py

```diff
@@ -24,7 +24,20 @@ def backfill_job(self, job: Union[Job, str]) -> str:
         task_id_to_backfill = job.taskcluster_metadata.task_id
         decision_job = job.fetch_associated_decision_job()
         decision_task_id = decision_job.taskcluster_metadata.task_id
-
+        side_by_side_task_id = ""
+
+        if "browsertime" in job.job_group.name:
```

Comment:
```suggestion
        if "Browsertime" in job.job_group.name:
```
Python is case sensitive and your check will not be valid since every name in the database starts with an uppercase letter.

Reply:
@alexandru-io There seems to be 33 job groups with the name containing 'Browsertime'. Could you check this list? Do we want to enable triggering `side-by-side` for each one that contains 'Browsertime'? I'm interested to know if there are any exceptions we should exclude.

Reply:
The context of this code is that sherlock backfills the performance jobs for which alerts were created. The side-by-side tool is designed to work only for recorded pageload jobs, for benchmarks and interactive for example doesn't work (exception is thrown). So giving the side-by-side is triggered only for what sherlock backfills, I think that the condition `"browsertime" in job.job_group.name` is enough.

Reply:
Browsertime Power Usage Tests on Fenix without fission enabled |  
-- | --
Browsertime performance tests on Fenix |  
Browsertime performance tests on Firefox |  
Browsertime performance tests on Firefox |  
Browsertime performance tests on Firefox |  
Browsertime performance tests on Firefox |  
Browsertime performance tests on Firefox (profiling) |  
Browsertime performance tests on Firefox with Gecko Profiling |  
Browsertime performance tests on Firefox with fission enabled |  
Browsertime performance tests on Firefox with populated bytecode cache. |  
Browsertime performance tests on Firefox with populated bytecode cache. (profiling) |  
Browsertime performance tests on Google Chrome Release


Browsertime performance tests on Google Chrome Release without fission enabled |  
-- | --
Browsertime performance tests on Google Chromium |  
Browsertime performance tests on Google Chromium |  
Browsertime performance tests on Safari |  
Browsertime performance tests on Safari |  
Browsertime performance tests on Safari |  
Browsertime performance tests on Safari |  
Browsertime performance tests on the reference browser |  
Browsertime power tests on Fenix |  
Browsertime power tests on Firefox without fission enabled |  
Browsertime power tests on Firefox/Geckoview |  
Browsertime power tests on reference browser |  
Browsertime tests on Chrome without fission enabled using live sites |  
Browsertime tests on Fenix without fission enabled |  
Browsertime tests on Fenix without fission enabled using live sites |  
Browsertime tests on Firefox with profiling and without fission enabled |  
Browsertime tests on Firefox without fission enabled |  
Browsertime tests on Firefox without fission enabled using live sites

Reply:
`"browsertime" in job.job_group.name` is incorrect because python is case sensitive and your check will not be valid since every name in the database starts with an uppercase letter as pointed out by your example ^

Reply:
check the updated code, please.